README.rst

```diff
@@ -6,71 +6,103 @@ Flake8 Spellcheck
 
 Flake8 Plugin that spellchecks variables, functions, classes and other bits of your python code.
 
-You can whitelist words that are specific to your project simply by adding them to ``whitelist.txt``
-in the root of your project directory. Each word you add  should be separated by a newline.
+You can whitelist words that are specific to your project simply by adding them to ``spellcheck-allowlist.txt``
+in the root of your project directory. Each word you add should be separated by a newline.
 
 Spelling is assumed to be in en_US.
 
 This plugin supports python 3.6+
```

Comment:
This note about Python version support is no longer accurate, right? I thought I saw that support for Python 3.6 has been dropped.

Reply:
Yep you need to rebase :)